jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -263,7 +312,7 @@ public JavaContainerBuilder addToClasspath(List<Path> otherFiles) throws IOExcep
         throw new NoSuchFileException(file.toString());
       }
     }
-    relativeClasspaths.add(OTHERS_CLASSPATH);
+    classpathOrder.add(LayerType.EXTRA_FILES);
```

Comment:
Huh, now I see we've been putting these "others" files into the `EXTRA_FILES` layer. When we get to use this class in our plugins, this won't fit. 1) Extra files in the plugins will go into the root, not `/app/classpath`; and 2) Plugins won't add a classpath for the extra files root (`/`). You will need to think about this divergence too.

Reply:
These aren't actually being added to the extra files layer, I'm just using the enum to keep track of the classpath order. Maybe this is a bit too hacky?

Reply:
I think we add "others" to the "extra files" layer.

```java
        layerConfigurationsBuilder.addDirectoryContents(
            LayerType.EXTRA_FILES, path, path1 -> true, appRoot.resolve(OTHERS_CLASSPATH));
```

Reply:
Let's keep this in mind. This will come to us soon.

Reply:
Oh yeah, you're right. They do get added to the extra layer, but `addToClasspath` isn't meant to be the same mechanism as the extra files configuration. So yeah, we'll probably have to find some other way to handle this.